src/script/components/InputLevel.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import cx from 'classnames';
+import {registerReactComponent} from 'Util/ComponentUtil';
+
+export interface InputLevelProps {
+  disabled: boolean;
+  level: number;
+}
+
+export const MAX_AUDIO_BULLETS = 20;
+
+/**
+ * Shows bullet indicators to visualize the audio input level.
+ *
+ * @param disabled - Don't show audio level indicator if set to `false`
```

Comment:
Comment is misleading since the component is still shown.

Reply:
Indeed. Changed it. :)